### DISCUSSION TITLE
Microsoft Cluster within Netbox

### DISCUSSION THREAD

**Discussion by tkn2023** [netbox-community/netbox#13314] [Help Wanted!]
Hi all,
I just wanted to ask wether there is an option to add virtual machines to a cluster as a host.
Since Microsoft Cluster is a thing we would like to make this visible within netbox.
The workaround would be to create a virtual machine and a "physical" Device to be able to add those to the cluster but the links to the "devices" would not be the correct once since the virtual machine is not a "device".
Any idea on how to make this work would be appreciated.
Thanks!

**Comment by hagbarddenstore** [comment]
Add a custom field on the Virtual Machine model to the Cluster model, "Hosts Cluster" or similar.

The Cluster model in NetBox is only meant to model hypervisor clusters, like VMware ESXi, Microsoft Hyper-V, Nutanix AHV, Proxmox, etc.

**Comment by candlerb** [comment]
For a similar purpose (lxd containers running inside KVM hosts), I create a "cluster" with no devices in it.

You can list the member VMs in the comment field of the cluster, or in a custom field as already suggested.
